.prettierrc.json

```diff
@@ -1,18 +1,4 @@
 {
```

Comment:
This file can be removed right ?

Reply:
I believe it can be removed but I am wondering why it is still in Morpho-Aave-V3 as we use the setUp for the formating. Do you have any idea @Rubilmax ?

Reply:
Honestly I don't, I believe it's an artifact